src/OrderFormAcceptanceTests.Objects/Pages/CompleteOrder.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using OpenQA.Selenium;
+using OrderFormAcceptanceTests.Objects.Utils;
+
+namespace OrderFormAcceptanceTests.Objects.Pages
+{
+    public sealed class CompleteOrder
+    {
+        public By FundingSourceYesContent => CustomBy.DataTestId("");
```

Comment:
I assume these will be updated in the future?

Reply:
yeah, looking at the FE tickets I don't think I can even pull the code and find it yet.